tests/ui/stats/test_stats_report.py

```diff
@@ -22,26 +22,39 @@
 
 import mock
 
+from rero_ils.modules.stats.api.api import Stat
 from rero_ils.modules.stats.api.report import StatsReport
 
 
-def test_stats_report_create(org_martigny):
+def test_stats_report_create(org_martigny, document):
     """Test the stat report creation."""
     cfg = {
+        "$schema": "https://bib.rero.ch/schemas/stats/stat-v0.0.1.json",
```

Comment:
Should be:
`"$schema": "https://bib.rero.ch/schemas/stats_cfg/stat_cfg-v0.0.1.json"`